Specs/DateTime/Hindi/DateTimePeriodParser.json

```diff
@@ -105,18 +101,18 @@
         "Start": 4,
         "Length": 31
       }
-    ]
+    ],
+    "NotSupported": "dotnet"
```

Comment:
Any specific reason not to support this case?
It's always helpful to add a "Comment" attribute to the spec case for documentation.

Reply:
no specific reason, this is still a work in progress and we have not finished solving all the cases yet.